Bottom: 7982a08529a8ce8ac729be3681af6d9683e44d70
Top:    5c2d2f23ac026c0e64fc87996522232d7800ccb2
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2014-01-01 01:21:06 -0500

Refresh of zarlink

---

diff --git a/package/kernel/linux/modules/sound.mk b/package/kernel/linux/modules/sound.mk
index 445763e..fc39ec7 100644
--- a/package/kernel/linux/modules/sound.mk
+++ b/package/kernel/linux/modules/sound.mk
@@ -301,7 +301,8 @@ $(eval $(call KernelPackage,sound-soc-rt5350))
 
 define KernelPackage/sound-soc-rt5350-le89156
   TITLE:=RT5350 SoC support for Zarlink LE89156
-  KCONFIG:=SND_SOC_RT5350_LE89156
+  KCONFIG:=\
+	SND_SOC_RT5350_LE89156
   FILES:=\
 	$(LINUX_DIR)/sound/soc/codecs/snd-soc-le89156.ko \
 	$(LINUX_DIR)/sound/soc/ralink/snd-soc-rt5350-le89156.ko
